swig/README.md

```diff
@@ -15,12 +15,15 @@ the `build` directory (see [DyNet documentation](http://dynet.readthedocs.io/en/
 
 ```
 build$ cmake .. -DEIGEN3_INCLUDE_DIR=../eigen -DINCLUDE_SWIG=ON
-build$ make
+build$ make dynet_swig && make
```

Comment:
It would be good to figure out why this is (sometimes) needed, the intention is that running just `make` should be sufficient.